src/components/Header.js

```diff
@@ -13,25 +14,37 @@ const HeaderWrapper = styled.header`
   padding: 0 80px;
 `;
 
-const NavWrapper = styled.nav`
+const Nav = styled.nav`
   display: flex;
   flex-direction: row;
   justify-content: flex-end;
   align-items: center;
+  margin-bottom: 2px;
 `;
 
+const NavLink = styled(Link)`
+  color: ${({ theme }) => theme.color.navigation};
+  text-decoration: none;
+  font-family: ${({ theme }) => theme.font.primary};
+  margin-left: 26.3px;
+`;
+
+NavLink.defaultProps = {
```

Comment:
Didn't see it until now, but the default props cause the theme object to be rendered into the DOM :smile: 
![image](https://user-images.githubusercontent.com/9722792/82787177-2bfbda80-9e66-11ea-9707-ee5293b77113.png)

Reply:
oops! haha